lib/fog/aws/requests/rds/describe_db_instances.rb

```diff
@@ -46,7 +46,6 @@ def describe_db_instances(identifier=nil, opts={})
              case server["DBInstanceStatus"]
              when "creating"
                if Time.now - server['InstanceCreateTime'] >= Fog::Mock.delay * 2
-                 region = "us-east-1"
                  server["DBInstanceStatus"] = "available"
                  server["AvailabilityZone"] ||= region + 'a'
```

Comment:
it's pulling from the services' region https://github.com/fog/fog-aws/blob/master/lib/fog/aws/rds.rb#L141